src/AppServiceFileSystem.ts

```diff
@@ -58,4 +60,8 @@ export class AppServiceFileSystem extends AzExtTreeFileSystem<FileTreeItem> {
         this._etags.set(node.fullId, etag);
         await node.refresh();
     }
+
+    public appendLineToOutput(value: string, options?: { resourceName?: string, date?: Date }): void {
+        ext.outputChannel.appendLog(value, options);
```

Comment:
Remove `appendLineToOutput` because it doesn't do anything and just use `ext.outputChannel.appendLog` directly